### ISSUE TITLE
Code style management

### ISSUE THREAD

**Issue by ryzhovalexandr** [APerricone/harbourCodeExtension#91]
In my team 4+ developers and legacy code base. I'd like to auto format code while editing (after save or select piece of code and press hot key) according our code style.
After integrate it in our pipline.
I think it could be very helpful not only for my team. Mayby it could be pretty easy to add to your extension, or create new one?
What do you think? I can try to contribute.

**Comment by ryzhovalexandr** [comment]
Yes, I have found it, thanks. In description of plug in there's nothing about it. What plans do you have about new features? I'd be very happy to see in plans
- register of reserve words (capital or lower case letters)
- local, params, private, public, static var naming. With vengarian notation and another prefixes
- use '==' and ':=' instead of '='

**Comment by APerricone** [comment]
It was an experiment, I planned to ask to some users to test it then finish it, but I never did it. So it is not advertised.
The use of Hungarian notation or other naming convention can be implemented as configurable extra warning, but is is not planned.
Actually all the extension is semi abandoned, starting next week I will not use harbour at work so probably I will not modify the extension anymore
